samcli/commands/local/lib/local_api_service.py

```diff
@@ -27,11 +27,14 @@ def __init__(self, lambda_invoke_context, port, host, static_dir):
         :param int port: Port to listen on
         :param string host: Local hostname or IP address to bind to
         :param string static_dir: Optional, directory from which static files will be mounted
+        :param tuple(string, string) ssl_context: Optional, path to ssl certificate and key files to start service
+            in https
```

Comment:
I'll look into this. I don't work in Python all that regularly, so `@dataclass` is new to me. But it would be great not to have to pass it around all over the place.